tiatoolbox/tools/__init__.py

```diff
@@ -0,0 +1,2 @@
+"""Package defines tools for the toolbox"""
+from tiatoolbox.tools import stainnorm, stainextract
```

Comment:
I know it's a very small file but missing the GPL header here.

Reply:
This was actually the case with **all** `__init__.py` files. Changed in [this](https://github.com/TIA-Lab/tiatoolbox/pull/35/commits/593da318650b992eee629b982adc3b0f4631a744) commit.